dataverse-webapp/pom.xml

```diff
@@ -142,16 +142,6 @@
             <artifactId>guava</artifactId>
             <version>27.0.1-jre</version>
         </dependency>
-        <!-- Should be refactored and removed once on Java EE 8 -->
-        <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-core</artifactId>
-        </dependency>
-        <!-- Should be refactored and removed once on Java EE 8 -->
-        <dependency>
-            <groupId>com.fasterxml.jackson.core</groupId>
-            <artifactId>jackson-databind</artifactId>
-        </dependency>
```

Comment:
Now I'm lost again... Why are those removed, instead of only changed into `provided` scope?

Reply:
Well since I added correct dependency in parent pom wouldn't having it again in child pom be redundant?

Reply:
No, you only added it in `<dependencyManagement>`, and this here is `<dependencies>` section. In `<dependencyManagement>` we don't actually declare dependencies, but only their versions and possibly other properties for easier re-use in modules. We still need them in `<dependencies>` to actually declare the dependency it a module that actually needs it.

Reply:
My bad, gotta read the docs.

Reply:
Btw do I need to add `provided` scope? Shouldn't it be inherited? `when a POM derived from this one declares a dependency described by a matching groupId and artifactId, the version and other values from this section are used for that dependency if they were not already specified.`

Reply:
You already defined the `scope` in `<dependencyManagement>` section, so you don't have to repeat it here. I usually only define versions there, as one might want to use the dependency in different scopes in different contexts. That's why I mentioned it in the first comment. So no, you don't need to :)